### DISCUSSION TITLE
office 365

### DISCUSSION THREAD

**Discussion by edwardgoodnow** [eleith/emailjs#337] [questions]
Same error no matter what options are used
SMTPError: connection encountered an error (0008A9C0BE7F0000:error:0A00010B:SSL routines:ssl3_get_record:wrong version number:../deps/openssl/openssl/ssl/record/ssl3_record.c:355

This is on a CentOs 8 server with openssl 3 installed have tried every setting metnioned including specifying tls cipher with v3 and tls set to false and everything in between

**Comment by eleith** [comment]
i haven't personally tested connecting to an [office365 smtp server](https://support.microsoft.com/en-us/office/pop-imap-and-smtp-settings-8361e398-8af4-4e97-b147-6c6c4ac95353), but i have connected to many other smtp servers using very similar settings (ex: port 587/starttls).

it is possible your error is either due to misconfiguration, or your centos was compiled with openssl in a way that it can't do a proper handshake with office365.
